docs/source/user-guide/sql/write_options.md

```diff
@@ -53,26 +53,18 @@ Finally, options can be passed when running a `COPY` command.
 ```sql
 COPY source_table
 TO 'test/table_with_options'
-(format parquet,
-compression snappy,
-'compression::col1' 'zstd(5)',
-partition_by 'column3, column4'
+OPTIONS (
+  format parquet,
+  compression snappy,
+  'compression::col1' 'zstd(5)',
+  partition_by 'column3, column4'
```

Comment:
Is this the correct format? Based on https://github.com/apache/arrow-datafusion/issues/9927  the `partition_by` has moved to the DML and it should be something like: `COPY t1 TO '/tmp/hive_output/' PARTITIONED BY (col1) OPTIONS (format parquet);`